backend/config/curation-api.yml

```diff
@@ -501,14 +501,15 @@ paths:
         "401":
           $ref: "#/components/responses/401"
 
-  /v1/collections/{collection_uuid}/asset:
-    get:
+  /v1/collections/{collection_uuid}/assets:
+    post:
```

Comment:
I think this should remain a `GET` since the underlying resource is not being changed in any way. Yes, we're creating additional download links each time, but those are ancillary to the actual entity (asset) in question.